### PR TITLE
Create and Mirror Jira Ticket

### PR DESCRIPTION
**Pull request by evisochek**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: (#14876)

## Description
This PR will merge the contribution PR (https://github.com/demisto/content/pull/14876)

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by guykeller** [comment]
Given that this PR is open, may I close [this](https://github.com/demisto/content/pull/14876) PR?

**Comment by evisochek** [comment]
> Given that this PR is open, may I close [this](https://github.com/demisto/content/pull/14876) PR?

Yes

**Comment by evisochek** [comment]
From teammate QA review the following issues came up:
Attachments - can not see any attachments coming from the Jira ticket to XSOAR
Add a Jira Key field so the user can search for it inside the Jira console
Add A project name or initials to the Jira issue status 
Label cannot be added through XSOAR
Created/ modified / Seen date are not mired through the integration 
Can not edit description and summary field in the XSOAR
Linked Issues \ sub tasks  cannot be seeing through the XSOAR panel.

@ssokolovich

File: Packs/Jira/IncidentFields/Jira_ID.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "Jira Incident",
+        "Jira Create Issue and Mirror"
+    ],
+    "associatedToAll": false,
+    "unmapped": false,
+    "unsearchable": false,
+    "caseInsensitive": true,
+    "sla": 0,
+    "threshold": 72,
+    "fromVersion": "5.5.0"
```
Comment (David-BMS) [inline Packs/Jira/IncidentFields/Jira_ID.json:34]: @evisochek New fields? why 5.5

File: Packs/Jira/IncidentFields/Jira_Issue_Key.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "Jira Incident",
+        "Jira Create Issue and Mirror"
+    ],
+    "associatedToAll": false,
+    "unmapped": false,
+    "unsearchable": false,
+    "caseInsensitive": true,
+    "sla": 0,
+    "threshold": 72,
+    "fromVersion": "5.5.0"
```
Comment (David-BMS) [inline Packs/Jira/IncidentFields/Jira_Issue_Key.json:34]: @evisochek New fields? why 5.5

File: Packs/Jira/Playbooks/playbook-Create_Jira_Issue.yml [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+name: Create Jira Issue
+description: "Create Jira issue allows you to open new issues.\nWhen creating the\
+  \ issue, you can decide to update based on on the issue's state, which will wait\
+  \ for the issue to resolve or close with StatePolling.\nAlternatively, you can select\
+  \ to mirror the Jira issue and incident fields.  To apply either of these options,\
+  \ set the SyncTicket value in the playbook inputs to one of the following options:\
+  \ \n1. StatePolling\n2. Mirror\n3. Leave Blank to use none\n\nWhen creating Jira\
+  \ issues through XSOAR, using the mirroring function, make sure that you exclude\
+  \ those issues when fetching incidents. To exclude these issues, tag the relevant\
+  \ issues with a dedicated label and exclude that label from the JQL query (Labels!=)."
```
Comment (David-BMS) [inline Packs/Jira/Playbooks/playbook-Create_Jira_Issue.yml:14]: @evisochek Maybe need to emphasize the JQL query defined in the integration instance settings

File: Packs/Jira/Playbooks/playbook-Create_Jira_Issue.yml [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+    \ StatePolling, and there is more than one Jira instance. It will specify the\
+    \ polling command to use a specific instance to run on. \nIf so, please add \"\
+    Using\" to the value. \nThe playbook will then take the instance name as the instance\
+    \ to use. "
+  playbookInputQuery:
+- key: InstanceName
+  value: {}
+  required: false
+  description: |
+    Set the Jira Instance that will be used for mirroring/running polling commands.
```
Comment (David-BMS) [inline Packs/Jira/Playbooks/playbook-Create_Jira_Issue.yml:490]: Consider adding
The instance name can be found in the integrations setting page
For example
jira-v2_instance_1

**Review by David-BMS — APPROVED** [review]
Please review comments before merging

File: Packs/Jira/IncidentFields/Jira_ID.json (hunk 1 above)
Comment (evisochek) [reply Packs/Jira/IncidentFields/Jira_ID.json:34]: updated

File: Packs/Jira/IncidentFields/Jira_Issue_Key.json (hunk 2 above)
Comment (evisochek) [reply Packs/Jira/IncidentFields/Jira_Issue_Key.json:34]: updated

**Pull request merged**